src/reducers/index.js

```diff
@@ -44,4 +45,5 @@ export default combineReducers({
   wizard: WizardReducer,
   fridge: FridgeReducer,
   breach: BreachReducer,
+  rowDetail: RowDetailReducer,
```

Comment:
do we like this name?

Reply:
we think we do. I can't think of a better  at the moment, and that's good enough for me.